lib/litani_report.py

```diff
@@ -299,16 +300,13 @@ def get_git_hash():
 
 
 def render(run, report_dir):
-    artifacts_dst = report_dir / "artifacts"
-    if artifacts_dst.exists():
-        shutil.rmtree(report_dir / "artifacts")
-    shutil.copytree(litani.get_artifacts_dir(), report_dir / "artifacts")
-    render_artifact_indexes(artifacts_dst)
+    temporary_report_dir = pathlib.Path(tempfile.gettempdir()) / str(uuid.uuid4())
```

Comment:
This is going to put the HTML directories under a temporary directory, where we won't later be able to find them. This will be a problem when we want to garbage-collect expired HTML directories.

I suggest you create an entirely new directory under the cache directory, perhaps `report_data`. So you'd have a method in `lib/litani.py` called `get_report_data_dir`. And you'd write these HTML directories to `get_report_data_dir() / str(uuid.uuid4())`.

That way, we can later simply iterate over all the HTML directories in there and delete them one by one (if they aren't locked). The cache directory is unique to a single running instance of litani, so it also means that several runs of litani won't garbage-collect each other's HTML directories.

So in summary: the cache directory would have a `html` and `report_data`. `html` at all times would be a symbolic link to one of the *subdirectories* of `report_data`.